app/src/main/java/com/macbitsgoa/events/aboutfest/OrganisersFragment.java

```diff
@@ -0,0 +1,69 @@
+package com.macbitsgoa.events.aboutfest;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+import com.macbitsgoa.events.R;
+
+import java.util.ArrayList;
+
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+public class OrganisersFragment extends Fragment {
+    private RecyclerView.Adapter adapter;
+    private ArrayList<OrganisersList> organisersList = new ArrayList<>();
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view;
```

Comment:
make all variables private by default

Reply:
Conflicts with codacy to keep the variable local in this case